mod_auth_vas4.c

```diff
@@ -1454,7 +1454,7 @@ static void auth_vas_server_init(apr_pool_t *p, server_rec *s)
 	    errinfo = vas_err_get_cause_by_type(sc->vas_ctx, VAS_ERR_TYPE_KRB5);
 
     	if (errinfo && errinfo->code == KRB5KDC_ERR_C_PRINCIPAL_UNKNOWN) {
-	        MAV_LOG_S(APLOG_INFO, s, "Credential test for %s failed with %s, " "this is harmless if it is a service alias", sc->server_principal, krb5_get_error_name(errinfo->code));
+	        MAV_LOG_S(APLOG_INFO, s, "Credential test for %s failed (principal is unknown), this is harmless if it is a service alias", sc->server_principal);
```

Comment:
ok, i will fix tomorrow

Reply:
I think we shouldn't, because krb5_get_error_name returned the string "KRB5KDC_ERR_C_PRINCIPAL_UNKNOWN" previously (see [1]). It seems to me that this branch displays a shortened error as log, because this case is harmless. The other if branch displays the full error including what errors it is caused by.

[1] https://dev.azure.com/1id/Privileged%20Access%20Management/_git/Authentication%20Services?path=%2FQAS%2FClient%2Fsrc%2Flibs%2Fapi%2Ferr.c&version=GBpbi-797136&line=294&lineStyle=plain&lineEnd=295&lineStartColumn=1&lineEndColumn=1